scripts/gen-descriptions.py

```diff
@@ -3,7 +3,7 @@
 import sys
 import xml.dom.minidom as md
 
-title_tmpl = u"{title-ios}"
+title_tmpl = u"{title-android}"
```

Comment:
Is this correct?

Reply:
Yes, now iOS only allows 30 char in the title. Will be renamed "title" soon